app/src/main/java/org/oppia/android/app/databinding/EditTextBindingAdapters.java

```diff
@@ -13,4 +13,9 @@
   public static void bindTextWatcher(@NonNull EditText editText, TextWatcher textWatcher) {
     editText.addTextChangedListener(textWatcher);
   }
+
```

Comment:
this is setting the error text to the layout. 
Not able to do from the XML side, doing here. 
Will add the kdoc.

Reply:
Done